examples/Creating_slides_with_the_Assistants_API.ipynb

```diff
@@ -0,0 +1,992 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Making slides with the Assistants API (GPT-4), and DALL·E-3\n",
```

Comment:
do you have a preference for "Making" vs "Creating" in both the title and name of the cookbook? I prefer "creating"